hw/Makefile

```diff
@@ -7,18 +7,23 @@ REG_OUTPUT_DV_DIR ?= ${REG_OUTPUT_DIR}/dv
 REG_OUTPUT_SW_DIR ?= ${REG_OUTPUT_DIR}/sw
 REG_OUTPUT_FPV_CSR_DIR ?= ${REG_OUTPUT_DIR}/fpv/vip
 
-IPS ?= uart          \
-       gpio          \
-       rv_plic       \
+IPS ?= aes           \
+       alert_handler \
        flash_ctrl    \
-       spi_device    \
-       rv_timer      \
+       gpio          \
        hmac          \
-       usbuart       \
-       alert_handler \
-       pinmux        \
+       i2c           \
+       nmi_gen       \
        padctrl       \
-       nmi_gen
+       pinmux        \
+       rv_plic       \
+       rv_timer      \
+       spi_device    \
+       trial1        \
+       uart          \
+       usb_fs_nb_pe  \
```

Comment:
I think this one can be removed.

Reply:
@vogelpi and @stefanlippuner are in the process of cleaning that IP up, so I think it is warranted to include it such that it always gets the newest regfile version... @vogelpi , WDYT?

Reply:
I think @sriyerg is right, this can be removed as usb_fs_nb_pe does not have a register file.

Is the idea of this PR to regenerate all register files with a single command to make sure nothing breaks?

Reply:
yes indeed. some of the modules where completely out of date. I came across them during my style linter cleanup, and since some cleanups are within the register templates, I had to regenerate them.

ah true, that module does not have a regfile, but the other usb module have. will remove it.